src/blog/interact-with-your-audience-with-a-live-poll.md

```diff
@@ -0,0 +1,70 @@
+---
+path: "/blog/interact-with-your-audience-with-a-live-poll"
+date: "2019-12-05"
+title: "Interact With Your Audience With A Live Poll"
+description: "We are introducing a new feature to let your interact with your audience without leaving your presentation"
+tags: "#webdev #javascript #opensource #speaking"
+image: "https://cdn-images-1.medium.com/max/1600/1*VST3XVmoX1R3hAXJ8OGw4A.jpeg"
+---
+
+![](https://cdn-images-1.medium.com/max/1600/1*ULPgfU4_6DDEHDLDCdUMKQ.jpeg)
+
+*Photo by [Nicholas Green](https://unsplash.com/@nickxshotz?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)*
+
+We had this idea for a long time and it took us a bit of time to realize it, that’s why we are so excited to introduce today the new feature of [DeckDeckGo](https://deckdeckgo.com/), our web open-source editor for presentations, to make possible the interaction with your audience through live polling without having to leave your slides 🔥
+
+<iframe width="280" height="158" src="https://www.youtube.com/embed/GuF58XBzTj0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe
+<br/>
+
+### User Experience
+
+UX could often be discussed and I’ve to say, in case of that particular feature, even more, as many solutions are feasible. I scratched my head for days, filling papers and taking notes until I finally figured out the core principles of the solution which were meaningful to me and hopefully to others too:
+
+* An audience could be a crowd, a small or large crowd. In case of live polling, not all participants are on the same page. Some might be fast to vote, some might be slow. Some people are near the projected slides or screen, others are probably faraway. Some might even be sleepy, some might have their phones in their bags, etc.
+
+* As a speaker I notably find convenient when I don’t have to switch between multiple applications, it could even to some extension makes me a bit less stressed. As an attendee, I tend to think that it needs me less effort to follow a presentation without to many media cut-offs. Summarized, I like when the presentation’s flow remains as smooth as possible.
```

Comment:
```suggestion
* As a speaker I find convenient when I don’t have to switch between multiple applications -- it could even to some extent makes me a bit less stressed. As an attendee, I tend to think that it's easier to follow a presentation without too many media interruptions. To sum it up, I like when the presentation’s flow remains as smooth as possible.
```